src/main/java/com/alrex/parcool/extern/shouldersurfing/ShoulderSurfingManager.java

```diff
@@ -1,37 +1,48 @@
 package com.alrex.parcool.extern.shouldersurfing;
 
 import com.alrex.parcool.common.action.impl.Dodge;
+import com.alrex.parcool.common.action.impl.Dodge.DodgeDirection;
 import com.alrex.parcool.extern.ModManager;
 import com.alrex.parcool.utilities.MathUtil;
 import com.github.exopandora.shouldersurfing.api.client.IShoulderSurfingCamera;
 import com.github.exopandora.shouldersurfing.api.client.ShoulderSurfing;
 import net.minecraft.client.Minecraft;
 import net.minecraft.client.player.LocalPlayer;
+import net.minecraft.world.phys.Vec3;
 
 public class ShoulderSurfingManager extends ModManager {
+    private static Minecraft mc = Minecraft.getInstance();
```

Comment:
This class can be accidently loaded from server side so I think we should not make a field of client only class.
Of course this should not be necessary if everything works fine but just I'm nervous of this kind crash.

Reply:
I can remove it and reference Minecraft.getInstance() directly from the methods to be safer. ShoulderSurfing methods may only be used on the client side, though, as they aim to manage camera angle.

Reply:
Yeah I think this behavior is better and I intended.

Reply:
Thank you very much. Then I'll fix what you pointed out an I'll try to get the vectors right for every camera angle